backend/src/main/java/com/github/clarakraus/susume/blog/Blog.java

```diff
@@ -25,6 +25,10 @@ public class Blog {
     private List <String> savedSusumes = new ArrayList<>();
     private List<String> friendsList = new ArrayList<>();
 
```

Comment:
Wieso braucht dein Blog denn das Passwort? Das sieht komisch aus und PasswortRepeat muss man eigentlich gar nicht speichern sondern nur einmal überprüfen, oder?

Reply:
i saw later, your Blog is your user object? but passwordRepeat shouldn't be necessary, just check it once